float64[] time
---
hmg_utils/BoxArray[] predictions
hmg_utils/ConvexHull[] static_obstacles